src/frontend/src/common/components/library/data_subview/components/IconButton/style.ts

```diff
@@ -0,0 +1,12 @@
+import styled from "@emotion/styled";
+import { getSpacings } from "czifui";
+
+export const StyledSpan = styled.span`
+  display: flex;
+  ${(props) => {
+    const spacings = getSpacings(props);
+    return `
+    margin-left: ${spacings?.m}px;
+  `;
```

Comment:
```suggestion
      margin-left: ${spacings?.m}px;
    `;
```

Reply:
interesting the linter didn't catch this